src/math/Matrix4.js

```diff
@@ -833,7 +833,7 @@ Matrix4.prototype = {
 
 	}(),
 
-	makeFrustum: function ( left, right, bottom, top, near, far ) {
+	makePerspective: function ( left, right, bottom, top, near, far ) {
```

Comment:
You don't want to change this to `left, right, top, bottom, near, far` so it matches `makeOrthographic`?